src/controllers/dlgprefcontroller.h

```diff
@@ -117,5 +117,6 @@ class DlgPrefController : public DlgPreferencePage {
     QSortFilterProxyModel* m_pInputProxyModel;
     ControllerOutputMappingTableModel* m_pOutputTableModel;
     QSortFilterProxyModel* m_pOutputProxyModel;
+    bool m_bGuiInitialized;
```

Comment:
Many other preferences pages call slotUpdate() during the constructor which actually initializes the GUI. Here we wait for the first slotUpdate signal, which happens later, I guess. Did you investigate the reason? If yes a comment would be helpful.

The different behaviours is probably one cause that this issue has happened. 
But, this is a good band aid fix anyway. Thank you. 

For me it would be more clear to rename the variable to something like.
m_initalUpdate; (without b, we have no convention for a b prefix btw).

Reply:
> Many other preferences pages call slotUpdate() during the constructor which actually initializes the GUI. Here we wait for the first slotUpdate signal, which happens later, I guess.

I'm afraid that pattern is not consistent and startup time is currently prolonged for no good reasons: unrelated/inadequate actions are triggered in `slotUpdate`, like writing to config, see #4667 (46b690e2b9ef69e0219e6c768c79b89431d146e4).
I'll continue cleaning up the preferences pages one by one.
In general, if `slotUpdate` is clean (only updates the GUI) it's called every time the Preferences are shown, so it doesn't make sense to me to call it during construction, that would only be repepetive.

I picked `m_GuiInitialized` so we can use
`if (m_GuiInitialized) {..`
instead the negation
`if (!m_InitialUpdate) {..`